k8s/environments/stg/common/juror-digital/jd-public.yaml

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: jd-public
+  namespace: jd
+  annotations:
+    flux.weave.works/automated: "false"
+    flux.weave.works/tag.chart-image: glob:2.*
+spec:
+  releaseName: jd-public
+  chart:
+    git: git@github.com:hmcts/sds-helm-charts
+    path: charts/jd-public
+    ref: master
+  values:
+    nodejs:
+      image: ssprivatestg.azurecr.io/jd/juror_digital_public_portal:2.21.427
+      ingressHost: 'jdfrontend-nle.platform.hmcts.net'
+      ingressSessionAffinity:
+        enabled: true
+        sessionCookieName: ""
```

Comment:
why does this need to be set like this? we can fix the chart upstream if required